vars/gitBranchName.groovy

```diff
@@ -1,3 +1,3 @@
 String call() {
-  sh(script: "git rev-parse HEAD | git branch -a --contains | sed -n 2p | cut -d'/' -f 3", returnStdout: true).trim()
+  sh(script: "git rev-parse HEAD | git branch -a --contains | grep '*' | cut -d ' ' -f 2", returnStdout: true).trim()
```

Comment:
Does this do what we want?
`git symbolic-ref HEAD|sed -e 's|refs/heads/||'`

Reply:
yes